api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -72,10 +72,10 @@
  *
  * <p>
  * If an LRA is propagated to a resource that is not annotated with any
- * particular LRA behaviour then the LRA will be suspended (ie the context
- * will not be available to the resource). But if this resource
- * then performs an outgoing JAX-RS request then the suspended LRA must be propagated
- * on this second outgoing request. For example, suppose resource <code>A</code>
+ * particular LRA behavior, then the value of the <code>mp.lra.LRA.value</code> configuration parameter
+ * determines how the LRA is propagated. The default of this parameter is <code>SUPPORTS</code> which
+ * means that the LRA must be propagated to outgoing requests.
```

Comment:
I agree with Martin's comment about using a boolean valued property here. If you agree then the property name needs changing.